### ISSUE TITLE
Specifying col and row for tiles only works for "small" tiles anything else will overlap

### ISSUE THREAD

**Issue by phantomotaku** [olton/Metro-UI-CSS#1133]
When I have multiple tiles of sizes medium,large, or wide in a tile-grid where the column and row values are specified the tiles will overlap.

Here is the link to the JS fiddle [https://jsfiddle.net/Lcqewurg/11/]
 
small and 'app' size tiles work as expected. If you see that I'm doing anything incorrectly can you let me know?

**Comment by olton** [comment]
https://metroui.org.ua/tiles.html#_tiles_grid

```
With tiles grid you can easy place tiles as you wish. The grid is divided into cells of size 70x70 with an interval of 10 px without limitation to height or width. You can limit the size of the grid with subclasses .size-* from 1 to 10. One size is equivalent of small tile. Sizes include gap.
```

**Comment by olton** [comment]
Fixed, Wait for 4.2.12 (tomorrow)
